README.md

```diff
@@ -137,31 +137,86 @@ All instructions going forward will assume that the environment variables have b
 
 ### Prepare Database
 
+```
```

Comment:
This markdown is incorrect, it's making Run be part of the markdown. I think you meant to say
Run:
```
make dev-prepare-database
```

Reply:
Resolved